Document Subscribe to trigger websocket (#1182)
home-assistant/developers.home-assistant: docs/api/websocket.md

}
```

## Subscribe to trigger

> To filter the results even more, you can subscribe to a trigger. With "subscribe_trigger" you can subscribe for just one trigger, this is just like a automation does it. For more info about the [trigger](https://www.home-assistant.io/docs/automation/trigger/).

Reviewer: ```suggestion
You can also subscribe to one or more triggers with `subscribe_trigger`. These are the same triggers syntax as used for [automation triggers](https://www.home-assistant.io/docs/automation/trigger/). You can define one or a list of triggers.
```

```json
{
    "id": 2,
    "type": "subscribe_trigger",

>     "trigger":     {

Reviewer: ```suggestion
    "trigger": {
```

        "platform": "state",
        "entity_id": "binary_sensor.motion_occupancy",
        "from": "off",
        "to":"on"
    },
}
```

As a response you get:

```json
{
 "id": 2,
 "type": "result",
 "success": true,
 "result": null
}
```

> For each trigger that matches, the server will send a message of type `trigger`. The `id` in the message will point at the original `id` of the `subscribe_trigger` command.

Reviewer: ```suggestion
For each trigger that matches, the server will send a message of type `trigger`. The `id` in the message will point at the original `id` of the `subscribe_trigger` command. Note that your variables will be different based on the used trigger.
```

```json
{
    "id": 2,
    "type": "event",
    "event": {
        "variables": {
            "trigger": {
                "id": "0",
                "idx": "0",
                "platform": "state",
                "entity_id": "binary_sensor.motion_occupancy",
                "from_state": {
                    "entity_id": "binary_sensor.motion_occupancy",
                    "state": "off",
                    "attributes": {

>                         "battery": 97,
>                         "illuminance": 189,
>                         "illuminance_lux": 189,
>                         "linkquality": 52,
>                         "occupancy": true,
>                         "temperature": 33,
>                         "voltage": 2995,

Reviewer: No need to specify more than necessary
```suggestion
```

                        "device_class": "motion",
                        "friendly_name": "motion occupancy"
                    },
                    "last_changed": "2022-01-09T10:30:37.585143+00:00",
                    "last_updated": "2022-01-09T10:33:04.388104+00:00",
                    "context": {
                        "id": "90e30ad8e6d0c218840478d3c21dd754",
                        "parent_id": null,
                        "user_id": null
                    }
                },
                "to_state": {
                    "entity_id": "binary_sensor.motion_occupancy",
                    "state": "on",
                    "attributes": {

>                         "battery": 97,
>                         "illuminance": 189,
>                         "illuminance_lux": 189,
>                         "linkquality": 52,
>                         "occupancy": true,
>                         "temperature": 33,
>                         "voltage": 2995,

Reviewer: ```suggestion
```
